app/src/main/java/com/eveningoutpost/dexdrip/SystemStatusFragment.java

```diff
@@ -262,13 +263,27 @@ private void setTransmitterStatus() {
         } else {
             transmitter_status_view.setText("" + td.sensor_battery_level);
             GcmActivity.requestSensorBatteryUpdate(); // always ask
-            if (td.sensor_battery_level <= Dex_Constants.TRANSMITTER_BATTERY_EMPTY) {
-                transmitter_status_view.append(" - very low");
-            } else if (td.sensor_battery_level <= Dex_Constants.TRANSMITTER_BATTERY_LOW) {
-                transmitter_status_view.append(" - low");
-                transmitter_status_view.append("\n(experimental interpretation)");
+            if (getBestLimitterHardwareName().equals("BlueReader")) {
```

Comment:
The classic system status tab is a legacy component which eventually will get removed once its features have been absorbed elsewhere. We shouldn't be adding new code here unless it is essential. I would suggest a better place for statistics and information display about bluereader is via the megastatus section in `DexCollectionService` somewhere like here: https://github.com/NightscoutFoundation/xDrip/blob/ab2a14277a98d171e0731db7e77901e02adb2a32/app/src/main/java/com/eveningoutpost/dexdrip/Services/DexCollectionService.java#L1275

Also it is usually only dexcom which has a transmitter battery as bluereader is a bridge, we would normally store that as a bridge battery. I haven't followed completely what you're doing here regarding the battery so perhaps I don't understand the purpose of this code. 

Data which you want to store persistently but which isn't actually a preference item. Eg we don't need to back it up from a configuration point of view and represents state etc should typically be placed in the `PersistentStore` There are of course places where xDrip historically uses preferences for this data which you may have copied the style from, but I would prefer if all new code uses the persistent store.